lib/grape-active_model_serializers/serializer_resolver.rb

```diff
@@ -7,23 +7,41 @@ def initialize(resource, options)
       end
 
       def serializer
-        @serializer ||= (
-          serializer_class.new(resource, options) if serializer_class
-        )
+        serializer_class.new(resource, serializer_options) if serializer_class
       end
 
       private
 
       attr_accessor :resource, :options
 
       def serializer_class
+        return @serializer_class if defined?(@serializer_class)
```

Comment:
This is interesting. Is it different from just checking `@serializer_class`? (aka `nil`)